sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1794,6 +1842,30 @@ def hpat_pandas_series_idxmax_index_impl(self, axis=None, skipna=True):
 @overload_method(SeriesType, 'mul')
 def hpat_pandas_series_mul(self, other, level=None, fill_value=None, axis=0):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.mul
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/mul.py
+       :language: python
+       :lines: 27-
+       :caption: Getting the multiplication of Series and other
+       :name: ex_series_mul
+
+    .. code-block:: console
+
+        > python ./series/mul.py
+        0      0.0
+        1      3.0
+        2    200.0
+        3      NaN
+        dtype: float64
+
+    Intel Scalable Dataframe Compiler Developer Guide
```

Comment:
Missing See Also section. Do not forget to use Pandas API Reference to understand how the final document should look like

Reply:
done